conf/application.conf

```diff
@@ -134,6 +134,11 @@ tracking-consent-frontend {
   gtm.container = "transitional"
 }
 
+contact-frontend {
```

Comment:
Just a check why we're adding this here? (Not sure if relevant to PR)

Reply:
This not being present is why the Technical Helper was not showing.

Reply:
Ok. Good work 👌